.github/workflows/docker-build-and-publish.yml

```
@@ -0,0 +1,32 @@
+name: Deploy Images to Github Container registry
+
+on:
+  push:
+
+jobs:
+  push-store-image:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Add tag for new version
+        id: semanticVersion
+        uses: anothrNick/github-tag-action@1.67.0
+        env:
+          DEFAULT_BUMP: patch
+          DRY_RUN: github.ref != 'refs/heads/main'
+          GITHUB_TOKEN: ${{ secrets.GIT_TOKEN}}
+      - name: 'Login to GitHub Container Registry'
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{github.actor}}
+          password: ${{secrets.GITHUB_TOKEN}}
+      - name: 'Build and publish image'
+        run: |
+          docker build . --tag local-seedcase-sprout
+          docker push local-seedcase-sprout ghcr.io/seedcase-project/seedcase-sprout:${{ steps.semanticVersion.outputs.tag }}-${GITHUB_SHA::7}
```

```suggestion
          
      - name: "Build and publish Docker image"
        uses: docker/build-push-action@v5
        with:
          context: .
          push: true
          tags: ${{ steps.meta.outputs.tags }}
          labels: ${{ steps.meta.outputs.labels }}
```

I'd rather use a standard action for doing this step, since it exists here with the `docker/build-push-action`.